views/mixins/_postForm.pug

```diff
@@ -1,5 +1,5 @@
 mixin postForm(post = {})
-		form#post(action=`/admin/new-post/${post.slug +'/' || ''}${post._id || '' }/edit` method="POST")
+		form#post(action=`/admin/new-post/${post.slug ? post.slug +'/' : ''}${ post._id ? post._id : '' }${ post.slug ? '/edit' : ''}` method="POST")
```

Comment:
Had to adjust this since we were ending up with `undefined`'s in the form action url